app/src/main/res/values/arrays.xml

```diff
@@ -44,4 +44,22 @@
         <item>@string/hk_host</item>
         <item>@string/akamai_host</item>
     </string-array>
+    <string-array name="tab_entries">
+        <item>直播</item>
+        <item>推荐</item>
+        <item>热门</item>
+        <item>番剧</item>
+        <item>影视</item>
+        <item>热点</item>
+        <item>其它</item>
+    </string-array>
+    <string-array name="tab_values">
+        <item>live</item>
```

Comment:
better to store uri's here

Reply:
it's not easy to store uri to a set (dict will be much better, if it could be used here) because i can't add a name to each item of a set, and i don't think `this == uriset.take(n) -> xxx` is a good idea

Reply:
i also tried store uri to stringset and use regex as tab_values ,then check if uri match any regex in the set, but in that way other tab filter will not work

Reply:
why the other tab won't work? i suppose to store uri/regex in entries, and for each uri we check if any of the uir/regex matches the uri.